VERSION

```diff
@@ -1 +1 @@
-4.1.0
+4.1.1
```

Comment:
```suggestion
4.1.1
```

See later comment - this has to be either a major or a minor bump (depending on which path we choose) - it is not a patch bump.

Reply:
I'll keep the client_signing_key and mark this as a minor change, as it involves new functionality